src/tests/bitsets/test_stdlib_bitset_64.f90

```diff
@@ -0,0 +1,744 @@
+        end if
+
+        call set0 % write_bitset( string0 )
+        if ( ('S33B' // bitstring_0) /= string0 ) then
+            error stop procedure // ' write_bitset_string failed to ' // &
+                'convert set0 value properly.'
+        else
+            write(*,*) 'write_bitset_string properly converted the set0 value'
+        end if
+
+        call set1 % write_bitset( string0 )
+        if ( ('S33B' // bitstring_all) /= string0 ) then
+            error stop procedure // ' write_bitset_string failed to ' // &
+                'convert set1 value properly.'
+        else
+            write(*,*) 'write_bitset_string properly converted the set1 value'
+        end if
+
+        return
+    end subroutine test_string_operations
+
+    subroutine test_io()
+        character(*), parameter:: procedure = 'TEST_IO'
+
+        integer :: unit
+
+        write(*,*)
+        write(*,*) 'Test bitset I/O: input, read_bitset, output, and ' // &
+            'write_bitset'
+
+        call set2 % from_string( bitstring_33 )
+        open( newunit=unit, file='test.txt', status='replace', &
+            form='formatted', action='write' )
+        call set2 % write_bitset(unit)
+        call set1 % write_bitset(unit)
+        call set0 % write_bitset(unit)
+        close( unit )
+        open( newunit=unit, file='test.txt', status='old', &
+            form='formatted', action='read' )
+        call set3 % read_bitset(unit)
+        call set5 % read_bitset(unit)
+        call set4 % read_bitset(unit)
+
+        if ( set4 /= set0 .or. set5 /= set1 .or. set3 /= set2 ) then
+            error stop procedure // ' transfer to and from units using ' // &
+                'bitset literals failed.'
+        else
+            write(*,*) 'Transfer to and from units using ' // &
+                'plain write_bitset_unit and read_bitset_unit succeeded.'
+        end if
+
+        close( unit )
+
+        open( newunit=unit, file='test.txt', status='replace', &
+            form='formatted', action='write' )
+        call set2 % write_bitset(unit, advance='no')
+        call set1 % write_bitset(unit, advance='no')
+        call set0 % write_bitset(unit)
+        close( unit )
+        open( newunit=unit, file='test.txt', status='old', &
+            form='formatted', action='read' )
+        call set3 % read_bitset(unit, advance='no')
+        call set4 % read_bitset(unit, advance='no')
+        call set5 % read_bitset(unit)
+
+        if ( set5 /= set0 .or. set4 /= set1 .or. set3 /= set2 ) then
+            error stop procedure // ' transfer to and from units using ' // &
+                ' bitset literals with advance == "no" failed.'
+        else
+            write(*,*) 'Transfer to and from units using ' // &
+                'write_bitset_unit and read_bitset_unit with ' // &
+                'advance=="no" succeeded.'
+        end if
+
+        open( newunit=unit, file='test.bin', status='replace', &
```

Comment:
I suggest to add some tests with `access='stream', form = 'unformatted'`

Reply:
OK